src/notification/Email-Service.js

```diff
@@ -0,0 +1,49 @@
+import config from '../config/notification/notification.config.js';
+import { MessageSender } from 'ffc-messaging';
+
+/**
+ * EmailService class responsible for sending submission emails and managing the connection to the message broker.
+ */
+export class EmailService {
+  /**
+   * Constructs an instance of EmailService.
+   * Initializes a message sender for sending emails using the configured contact details queue.
+   */
+  constructor() {
+    this.sender = new MessageSender(config.contactDetailsQueue);
```

Comment:
Would prefer this to be injected allowing us to swap out the protocol.

Reply:
Agreed. Let's chat.